app/src/main/java/org/sopt/cgv/feature/seats/component/SeatSelectionTimeCardRow.kt

```diff
@@ -0,0 +1,110 @@
+package org.sopt.cgv.feature.seats.component
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+import org.sopt.cgv.core.designsystem.component.card.CompTimeCard
+
+data class TimeCardContent(
+    val startTime: String,
+    val endTime: String,
+    val currentSeats: Int,
+    val totalSeats: Int,
+    val isMorning: Boolean,
+    val isActivated: Boolean
+)
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun SeatSelectionTimeCardRow(
+    contents: PersistentList<TimeCardContent>,
+    viewModel: SeatSelectViewModel,
```

Comment:
```suggestion
    selectedIndex: Int,           
    onCardClick: (Int) -> Unit,   
```
여기서도 ViewModel 직접 참조 대신 선택된 인덱스(selectedIndex)와 클릭 이벤트 콜백(onCardClick)을 매개변수로 받도록 변경하여 상태 관리의 책임을 상위 컴포넌트로 이동시키게 된다면 컴포넌트의 재사용성 향상시킬 수 있을 거 같아요! 또한 컴포넌트가 특정 ViewModel에 종속되지 않아 더 다양하게 사용 가능할 거 같습니당